dencam/gui.py

```diff
@@ -66,7 +66,7 @@ def _setup(self):
 
             frame.grid(row=0, column=0, sticky='nsew')
 
-        self.show_frame('RecordingPage')
+        self.show_frame('NetworkPage')
```

Comment:
Are you making this change because it matches better where the system immediately goes upon the first run of _update?  I suspect it is thus a non-functional change because of the way the code is structured but it might still be a good refactor to make the code more straight-forward i.e. communicates where the GUI is actually meant to start. I am just interested in your thinking.